### DISCUSSION TITLE
A faster json parsing library.

### DISCUSSION THREAD

**Discussion by ZeyadMoustafaKamal** [encode/django-rest-framework#9465] [Ideas & Suggestions] [answered]
I just saw [orjson](https://github.com/ijl/orjson), which is faster than the built-in JSON library. so I would recommend using it as it will not require a lot of modifications.

**Comment by onegreyonewhite** [comment, marked as answer]
@ZeyadMoustafaKamal just use [drf_orjson_renderer](https://github.com/brianjbuck/drf_orjson_renderer) for that.

**Comment by arminshfatemi** [comment]
@onegreyonewhite 
hi does the suggested package is production ready ??

**Reply by onegreyonewhite** [reply]
@arminshfatemi It depends on what you mean by that. But in 3 years of use, I’ve never encountered any issues.

Both the parser and the renderer are ridiculously simple, so there’s practically nothing to break.

The only thing is that it’s stricter with the int type, so it might throw an error for excessively large integers. But that’s at the level of orjson itself.
